sqlmesh/core/environment.py

```diff
@@ -16,6 +16,10 @@
 T = t.TypeVar("T", bound="EnvironmentNamingInfo")
 
 
+SnapshotTableInfoOrJsonList = t.Union[t.List[t.Dict], t.List[SnapshotTableInfo]]
```

Comment:
this says json list, but you have a t.Dict, isn't that confusing?

Reply:
should you change to Union[t.List[str], t.List[SnapshotTableInfo]]

Reply:
ok, so seems like you want dict, can we rename to

tableinfOrDicts

Reply:
OBE